### PR TITLE
Pricing cards

### PR DESCRIPTION
**Pull request by vinc3nati**
## Related Issue
  - This issue solves the aged way of displaying pricing plan cards by adding user experience and interactivity.

Closes: #914  

### Describe the changes you've made
- I have added  "PricingCards.html" --> public  
- "pricingCards.css" --> css 
- "PricingCards" folder --> Images 
- Updated "script.js"

### Describe if there is any unusual behaviour of your code(Write NA if there isn't)
NA

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
**Front Side**
> image

**Popup**
> image

### CODE REVIEW

**Review by urvashi-code1255 — COMMENTED** [review]
@vinc3nati You haven't added your project's GIF in assets folder.

**Comment by vinc3nati** [comment]
> @vinc3nati You haven't added your project's GIF in assets folder.

Yes I have updated it's in the folder <PricingCards> in the GIFs section.

**Comment by vinc3nati** [comment]
> @vinc3nati You haven't added your project's GIF in assets folder.

Hello there , I have updated the GIFs folder with the project's gif file. Hope everything is fine now.

**Comment by urvashi-code1255** [comment]
@vinc3nati Haven't you used any Js in your project?

**Comment by vinc3nati** [comment]
> @vinc3nati Haven't you used any Js in your project?

Yes I had planned on using it but sass made things easy for me.
Nonetheless if you want me to add js Lemme know I'll do the same.

**Comment by urvashi-code1255** [comment]
@vinc3nati As this website is for vanilla js projects. I request you to do this with vanilla js.

**Comment by vinc3nati** [comment]
> @vinc3nati As this website is for vanilla js projects. I request you to do this with vanilla js.

Roger that.

**Comment by vinc3nati** [comment]
Hey @urvashi-code1255 Vanilla JS has been added as you asked for.

**Comment by vinc3nati** [comment]
Hey @urvashi-code1255 is there any problem with my pull request because it's been pending since 2 days?

File: script.js [hunk 1]
```diff
@@ -907,10 +907,17 @@ let projectData = [
     projectUrl: "public/rose-mathematics.html",
   },
   {
+<<<<<<< HEAD
+    projectName: "Pricing Plan Cards",
+    projectImage: "assets/GIFs/PricingCards.gif",
+    projectUrl: "public/PricingCards.html",
+  }
+=======
```
Comment (991rajat) [inline script.js:915]: Remove this merge conflict tags.

**Review by 991rajat — CHANGES_REQUESTED** [review]
Reduce the size of the gif please.
If not able to reduce replace with the static image.

File: script.js (hunk 1 above)
Comment (vinc3nati) [reply script.js:915]: Roger that

**Comment by vinc3nati** [comment]
> Reduce the size of the gif please.
> If not able to reduce replace with the static image.

On it

**Comment by vinc3nati** [comment]
Reduced the size of  the gif and fixed the merge conflict. @urvashi-code1255 @991rajat

**Review by 991rajat — APPROVED** [review]
Looks Great.

**Comment by 991rajat** [comment]
Thanks for this awesome contribution. 🤩 
Keep Contributing.

**Comment by 991rajat** [comment]
Learn about squashing the commits. It's a good practice.

**Comment by vinc3nati** [comment]
Gotcha.

**Pull request merged**